src/main.mjs

```diff
@@ -2,37 +2,41 @@ import { google } from 'googleapis';
 import { WebClient } from '@slack/web-api';
```

Comment:
Heads up, moved this to a mjs file instead of a ts file so I didn't need to transpile it.

I don't think typescript is giving us a lot in terms of functionality here.

Reply:
@cstebbinsflex I like what you've done!

Reply:
What do you mean by functionality?